### DISCUSSION TITLE
Unable to join Discord

### DISCUSSION THREAD

**Discussion by blackmann** [motion-canvas/motion-canvas#772] [General]
I have been attempting to join the motion canvas discord, but I can't join. Checking the logs in the browser, it returns 403 Forbidden.

I made a friend who's in there send me an invite and I get this response:

![image](https://github.com/motion-canvas/motion-canvas/assets/5024388/7602fdfe-f114-4cb1-88fc-440c65323546)

I have barely ever interacted in the server... I can't understand why I'm banned (if this is not a mistake).

@aarthificial

**Comment by aarthificial** [comment]
> I have barely ever interacted in the server

Your account has never interacted in the server.
Three days after joining, it was used to post advertisements in the offtopic channel.

**Reply by blackmann** [reply]
I'm very sorry if I actually did post an ad... I have no recollection of this.

Is the ban permanent or I have to wait for some time? I wouldn't try to appeal if I intended to be malicious.

**Reply by blackmann** [reply]
Oh, I haven't shared my Discord username yet, how did you know I posted an ad?

**Reply by aarthificial** [reply]
> how did you know I posted an ad?

You use the same profile picture on Discord:
![image](https://github.com/motion-canvas/motion-canvas/assets/64662184/0dbb8b6f-7584-4a75-87fe-bf2ca0b6d5e1)

What you posted was a promo for stylebender so it's weird that you have no recollection of it.
I revoked the ban but please know that joining a server and then immediately promoting your own projects without interacting even once paints a very bad picture.

**Reply by blackmann** [reply]
Thanks @aarthificial 

Yes, stylebender is me. I'm sorry about that. 

Noted!
